### ISSUE TITLE
Chinese characters in fields while attempting to create a New Encrypted Vault

### ISSUE THREAD

**Issue by gituzzer** [iterate-ch/cyberduck#13557]
Version: Cyberduck 8.4.1 (38006)

Bug: On a MS onedrive connetion (which had 3 My Files, 2 or them normal folders and another an encrypted vault), I tried to create an additional encrypted vault.
But  as I opened the dialog box to Create the vault, chinese/korean? characters show up inside the edit fields. I tried to delete them but they move between the fields, and I cannot create the Encrypted Vault.
See attached pic.

![bandicam 2022-07-09 21-43-52-597](https://user-images.githubusercontent.com/39234054/178122050-0972793c-41c9-4c2a-90c8-1a492cf560f3.jpg)

Thanks for any tip on  how to solve this.
(U

**Comment by gituzzer** [comment]
Worked around it, looks like it was just garbage text, and after checking Help, I checked fields are the password and password-confirm (labels are missing as you see), so I entered the same password and it created the vault fine.
1 - Force-clear the fields upon opening the dialog. String.Empty(..) or whatever.
2 - Add password static texts "Enter password", "Confirm password" to the password fields.
Anyway I think i'll close it as the function itself works, just not the GUI.
